### PR TITLE
app-text/pelican: fix tests on musl

### PR DESCRIPTION
**Pull request by matoro**
These fixes come from [Alpine repo](https://git.alpinelinux.org/aports/tree/testing/py3-pelican).

### CODE REVIEW

**Comment by thesamesam** [comment]
matoro, could you rebase please?

also, @hartwork, could you take a look? (I figure you have some interest in it given touched it recently, better than me doing it blindly; I can do it if you have no interest in the pkg though, just figured I'd ask) thanks!

**Comment by matoro** [comment]
> matoro, could you rebase please?
> 
> also, @hartwork, could you take a look? (I figure you have some interest in it given touched it recently, better than me doing it blindly; I can do it if you have no interest in the pkg though, just figured I'd ask) thanks!

Done, updated title to reflect.

File: app-text/pelican/pelican-4.9.1.ebuild [hunk 1]
```diff
@@ -56,6 +56,17 @@ EPYTEST_DESELECT=(
 
 distutils_enable_tests pytest
 
+python_prepare_all() {
+	# bug 863962
+	if use elibc_musl
```
Comment (thesamesam) [inline app-text/pelican/pelican-4.9.1.ebuild:61]: Let's [avoid conditional patching](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching) if we can.

It's only a handful which failed - we could `EPYTEST_DESELECT` all of those, or we could patch in something like GENTOO_SKIP_LOCALE_TESTS to `locale_available` and set that for musl. I'd probably prefer the former.

Comment (matoro) [reply app-text/pelican/pelican-4.9.1.ebuild]: Sure, done.

**Pull request closed without merging**